src/plugins/maps-dashboards-plugin/public/common/opensearch_maps_client.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { EMSClient } from '@elastic/ems-client';
+
+export class OpenSearchMapsClient extends EMSClient {
+  constructor({ osdVersion, manifestServiceUrl, language, landingPageUrl, fetchFunction }) {
+    super({ osdVersion, manifestServiceUrl, language, landingPageUrl, fetchFunction });
+    this._queryParams = {
+      osd_version: osdVersion,
+      opensearch_tos_agree: true,
+    };
+    this._manifestServiceUrl = manifestServiceUrl;
+  }
+
+  async isEnabled() {
+    let result;
+    try {
+      result = await this._fetchWithTimeout(this._manifestServiceUrl);
+    } catch (e) {
+      // silently ignoring the exception and returning false.
+      return false;
```

Comment:
Based on last bug fix on dashboards repo, here we want to return true. https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1550/files#diff-30df0a18dfe7348e10bec7d9782208a172c77771c113061b47de51a25b22f2cd